### PR TITLE
handle page and screen without name calls

### PR DESCRIPTION
**Pull request by sayan-mitra**
## Description of the change

> Description here

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: v0/destinations/indicative/data/INScreenConfig.json [hunk 1]
```diff
@@ -11,10 +11,11 @@
   },
   {
     "destKey": "eventName",
-    "sourceKeys": "event",
-    "required": true,
+    "sourceKeys": "name",
```
Comment (arnab-p) [inline v0/destinations/indicative/data/INScreenConfig.json:14]: Let's use `["name", "properties.name"]`

File: v0/destinations/indicative/data/INScreenConfig.json [hunk 2]
```diff
@@ -11,10 +11,11 @@
...
+    "required": false,
     "metadata": {
-      "template": "Screen View"
+      "defaultValue": "Screen View",
```
Comment (arnab-p) [inline v0/destinations/indicative/data/INScreenConfig.json:17]:
```suggestion
      "defaultValue": "Screen Viewed",
```

File: v0/destinations/indicative/data/INPageConfig.json [hunk 3]
```diff
@@ -12,9 +12,10 @@
   {
     "destKey": "eventName",
     "sourceKeys": "name",
```
Comment (arnab-p) [inline v0/destinations/indicative/data/INPageConfig.json:14]: Let's use `["name", "properties.name"]`

For safety

File: v0/destinations/indicative/data/INPageConfig.json [hunk 4]
```diff
@@ -12,9 +12,10 @@
...
-    "required": true,
+    "required": false,
     "metadata": {
-      "template": "Page View"
+      "defaultValue": "Page View",
```
Comment (arnab-p) [inline v0/destinations/indicative/data/INPageConfig.json:17]:
```suggestion
      "defaultValue": "Page Viewed",
```

**Review by arnab-p — CHANGES_REQUESTED** [review]

File: v0/destinations/indicative/data/INPageConfig.json (hunk 4 above)
Comment (sayan-mitra) [reply v0/destinations/indicative/data/INPageConfig.json:17]: keeping it as discussed

File: v0/destinations/indicative/data/INScreenConfig.json (hunk 2 above)
Comment (sayan-mitra) [reply v0/destinations/indicative/data/INScreenConfig.json:17]: keeping it as discussed

File: v0/destinations/indicative/data/INScreenConfig.json (hunk 1 above)
Comment (sayan-mitra) [reply v0/destinations/indicative/data/INScreenConfig.json]: done

File: v0/destinations/indicative/data/INPageConfig.json (hunk 3 above)
Comment (sayan-mitra) [reply v0/destinations/indicative/data/INPageConfig.json]: done

**Review by arnab-p — APPROVED** [review]

**Pull request merged**